scripts/variantstore/wdl/GvsExtractCohortFromSampleNames.wdl

```diff
@@ -0,0 +1,84 @@
+version 1.0
+
+import "GvsPrepareCallset.wdl" as GvsPrepareCallset
+import "GvsExtractCallset.wdl" as GvsExtractCallset
+
+# Workflow used by AoU to extract variants for a given cohort of sample_names
+
+workflow GvsExtractCohortFromSampleNames {
+
+  input {
+    File cohort_sample_names
+    String query_project
+    String gvs_project
+    String gvs_dataset
+    String fq_gvs_extraction_cohorts_dataset
+    String fq_gvs_extraction_destination_dataset
+    String fq_gvs_extraction_temp_tables_dataset
+    String extraction_uuid
+    String? output_gcs_dir
+
+    # Extract parameters
+    File wgs_intervals
+    Int scatter_count
+
+    File reference
+    File reference_index
+    File reference_dict
+
+    String output_file_base_name
+
+    Boolean do_not_filter_override = false
+    String? filter_set_name
+    String fq_filter_set_info_table = "~{gvs_project}.~{gvs_dataset}.filter_set_info"
+    String fq_filter_set_site_table = "~{gvs_project}.~{gvs_dataset}.filter_set_sites"
+    String fq_filter_set_tranches_table = "~{gvs_project}.~{gvs_dataset}.filter_set_tranches"
```

Comment:
I don't think you need to set these values or pass them to the workflow since they are defaulted in the workflow. If you want to allow the caller to set them, you should make them optional and then you would pass them into the workflow.